tools/uw-frame-static/build.js

```diff
@@ -12,9 +12,12 @@ var exec_handler = function(error, stdout, stderr) {
 
 var mkdirp = require('mkdirp');
 var copy = require('recursive-copy');
+var copyOptions = {
```

Comment:
yup. and we're using const elsewhere in the file.  I'll update the file to use it consistently.